.changeset/nice-rocks-kick.md

```diff
@@ -0,0 +1,18 @@
+---
+'@whatwg-node/server-plugin-cookies': patch
```

Comment:
Only this package is enough. Could you remove the others from the changeset? Thanks!

Reply:
Changeset updated

Reply:
Thank you 🙏